content/en/docs/reference/glossary/limitrange.md

```diff
@@ -0,0 +1,23 @@
+---
+title: LimitRange
+id: limitrange
+date: 2019-04-15
+full_link: 
+short_description: >
+  Provides constraints to limit resource consumption per Containers or Pods in a namespace.
+
+aka: 
+tags:
+- core-object
+- fundamental
+- architecture
+related:
+ - pod
+ - container
+
+---
+ Provides constraints to limit resource consumption per {{< glossary_tooltip text="Containers" term_id="container" >}} or {{< glossary_tooltip text="Pods" term_id="pod" >}} in a namespace.
```

Comment:
I'm concerned about the number of glossary entries we're marking as “fundamental”.

Because of that concern, I'd be tempted to place this definition either in the page https://kubernetes.io/docs/concepts/policy/resource-quotas/ or in a new page that's a sibling to it. Then we can rely on the quite good website search function to let visitors find that information.

What do you think @nelvadas?

Reply:
I do not have a strong opinion on this @sftim, it makes sense to have a seperate page  as   https://kubernetes.io/docs/concepts/policy/resource-quotas/ in the policy folder but also to have the term in the glossary.
At the moment we have this page https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/  with a section on limitrange.

Reply:
Would it make sense to put a link to that section in the `full_link` metadata for the glossary entry?

Reply:
Right now I couldn't allocate time to making a new page about LimitRange. I'll track it as something I might pick up at some point.

Reply:
Ok @Sftim, I am working on it and will let you know.